doc/release-notes/upgrading.en.rst

```diff
@@ -180,6 +180,7 @@ The following :file:`records.yaml` changes have been made:
 - The records.yaml entry :ts:cv:`proxy.config.allocator.hugepages` will enable
   allocating iobuffers and cache volumes from hugepages if configured in the
   system.
+- The records.yaml entry ``proxy.config.plugin.compiler_path`` has been added to specify an optional compiler tool path for compiling plugins.
```

Comment:
Is it worth mentioning this is for cripts?

Reply:
We could, but I left it a bit vague on purpose, following how Leif wrote his description of the config in his PR, since he never mentions the cripts in it.